node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -916,12 +916,13 @@ impl Initialized {
 			} else {
 				self.metrics.on_queued_best_effort_participation();
 			}
+			let request_timer = Arc::new(self.metrics.time_participation_pipeline());
 			let r = self
 				.participation
 				.queue_participation(
 					ctx,
 					priority,
-					ParticipationRequest::new(new_state.candidate_receipt().clone(), session),
+					ParticipationRequest::new(new_state.candidate_receipt().clone(), session, request_timer),
```

Comment:
My first reaction was maybe `request_timer` can be initialized in `ParticipationRequest::new`. But after looking at it, doing it like this instead of hiding it inside `ParticipationRequest::new` makes it more explicit that we are starting a timer here. 👍